app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -177,7 +177,7 @@ def max_rated_disabilities_from_ipf
     rated_disabilities = fd['rated_disabilities'] || []
 
     rated_disabilities.select do |dis|
-      dis['maximum_rating_percentage'] == dis['rating_percentage']
+      !dis['maximum_rating_percentage'].nil? && dis['maximum_rating_percentage'] == dis['rating_percentage']
```

Comment:
Ruby style prefers to avoid negations when an antonym method is available. Maybe `.present?` could work here.